stable/defaultstorageclass/README.md

```diff
@@ -0,0 +1,4 @@
+# defaultstorageclass
+
+Webhooks and controllers relating to default storage classes.
```

Comment:
Can you explain what it does as a chart ?

Reply:
my bad. added.